plugin/src/main/kotlin/com/chimerapps/storageinspector/ui/ide/view/sql/CustomDataTableView.kt

```diff
@@ -338,7 +338,16 @@ private class TableViewColumnInfo(
     }
 
     private fun parseDateTime(raw: String): Long {
-        return OffsetDateTime.parse(raw).toInstant().toEpochMilli();
+        // If the DateTime ends with a Z or a time zone (e.g. +00:00), parse it
+        // If not, assume that it is in UTC (per SQLite convention) and append a Z before parsing it
+        val pattern = Pattern.compile(".*(\\+\\d{2}:\\d{2}|Z)$");
```

Comment:
This will transform 2023-10-03T12:00:00.000+0 to 2023-10-03T12:00:00.000+0Z which feels wrong too

Reply:
The new version I just pushed will always return either a valid ISO string, or return `null`. I got the `runIde` task working too, and from my testing the cell will be empty when the timestamp could not be parsed (`null` is returned from `parseDateTime`)

Reply:
2023-10-03T12:00:00.000+0 is now just considered a wrongly formatted datetime, and `null` is returned s.t. it won't be displayed